app/views/claims/_check_your_answers_section.html.erb

```diff
@@ -16,7 +16,7 @@
       </dd>
 
       <dd class="govuk-summary-list__actions">
-        <%= link_to('Change', claim_path(current_policy_routing_name, slug), class: 'govuk-link') unless slug.nil? %>
+        <%= link_to('Change' , claim_path(current_policy_routing_name, slug), class: 'govuk-link', 'aria-label': "Change  #{label.downcase}") unless slug.nil? %>
```

Comment:
Are the extra spaces significant to the change, or did they sneak in when removing the extra words? I'm not up to speed with how screen-readers may interpret them so may be entirely legit!

Also, an errant space has snuck in after the first parameter of `link_to()` which could probably be removed in the interests of cleanliness.

Reply:
Thanks Nick, nope those spaces shouldn't have been in there, I've amended it now :)